code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -418,7 +418,8 @@ GLOBAL_LIST_EMPTY(lifts)
 				var/datum/callback/land_slam = new(collided, TYPE_PROC_REF(/mob/living/, tram_slam_land))
 				collided.throw_at(throw_target, 200 * collision_lethality, 4 * collision_lethality, callback = land_slam)
 
-				SEND_SIGNAL(src, COMSIG_TRAM_COLLISION)
+				if(istype(collided, /mob/living/carbon/human)) //don't increment the counter when birdboat gets hit
+					SEND_SIGNAL(src, COMSIG_TRAM_COLLISION)
```

Comment:
dont gate the signal like this, check for this on the registered proc

Reply:
done